irods/client_configuration/__init__.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import print_function
+import ast
+import copy
+import io
+import logging
+import os
+import re
+import sys
+import types
+
+logger = logging.Logger(__name__)
+
+class iRODSConfiguration(object):
+    __slots__ = ()
+
+def getter(category, setting):
```

Comment:
What is `getter`? `getter` feels very broad. Is there a better name we can use to better express the intent?

Is this used?

Reply:
`getter` is used by the `data_object_manager` **open** function - currently there only.   Its eventual, general purpose will be  as a utility for that and other library routines which need to know the current setting of a config line-item.

Reply:
I see. I think it's fine as is.

A comment explaining that would be helpful for future readers though.

Reply:
Comment added.